buddyboss/buddyboss-platform
# #1833  - fixed: not showing update nickname on group discussion

### All Submissions:






### Changes proposed in this Pull Request: 



Fixes https://github.com/buddyboss/buddyboss-platform/issues/1833 .

### How to test the changes in this Pull Request:

1. Create a new group with discussion and add members to this group
2. Change the nickname of a member of this group
3. Go to Group > Discussion
4. Create a new topic in this Discussion.
5. Click on the reply button and mention the user who changed the nickname.
6. Nickname will appear in the auto-populated list

### Proof Screenshots or Video
https://vimeo.com/577125153/94ef59e0c9



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
